### PR TITLE
Add OpenFF Aniline 2D Impropers v1.0

### PR DESCRIPTION
**Pull request by SimonBoothroyd**
- [x] Created a new folder in the submissions directory containing the dataset
- [x] Added README.md describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [x] All files used to produce the dataset are included with a description
- [x] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset is named `dataset.json`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [ ] Make a new dataset entry in the mapping table, INDEX.md
- [x] Ready to submit!

### CODE REVIEW

**Comment by SimonBoothroyd** [comment]
@jmaat - the TODOs are:

* Update the notebook to add a description and tag line (currently these say "TODO TODO")
* Run the notebook and commit the dataset.* files and push them here.
* Update the description in the README file and copy and paste the notebook output into the `General Information` and `Metadata` sections.

**Comment by jmaat** [comment]
hey @SimonBoothroyd, I did those tasks for the PR

**Review by SimonBoothroyd — COMMENTED** [review]
Thanks @jmaat - those look great! I realised I missed a comma when pasting in the SMILES patterns which lead to two patterns getting joined together, and also forget to update the title - both these things should have been fixed in b3411bd.

@trevorgokey @jthorton @dotsdl this should be ready for review

**Comment by SimonBoothroyd** [comment]
This PR should remain un-merged until #200 has completed and been analysed

**Comment by dotsdl** [comment]
Not starting this until after #201 as well.

**Comment by SimonBoothroyd** [comment]
@dotsdl can we get this one running with medium priority?

**Review by jthorton — APPROVED** [review]

**Comment by dotsdl** [comment]
Reviewing!

File: submissions/2021-03-29-OpenFF-Aniline-2D-Impropers-v1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ - Enumerate protomers: False
+
+
+### QCSubmit generation pipeline
+
+ - `generate-dataset`: The notebook used to generate the dataset from the input smiles set.
+
+### QCSubmit Manifest
+
+- `generate-smiles.py`: Molecule generation script.
```
Comment (dotsdl) [inline submissions/2021-03-29-OpenFF-Aniline-2D-Impropers-v1.0/README.md:36]: I don't see this file; is it committed? If the SMILES were hand-selected, won't be necessary.

**Review by dotsdl — APPROVED** [review]
Looks excellent @SimonBoothroyd and @jmaat! 

Just one minor comment: please feel free to commit the `generate-smiles.py` if one was used to generate SMILES, but it appears they were hand selected so likely not necessary.

**Comment by SimonBoothroyd** [comment]
@dotsdl can we remove this from error cycling?

**Pull request merged**